driver/ni/ni.h

```diff
@@ -25,448 +25,434 @@
 #include "client/cpp/synnax.h"
 #include "driver/task/task.h"
 #include "driver/pipeline/acquisition.h"
-#include "driver/errors/errors.h" 
+#include "driver/errors/errors.h"
 #include "driver/breaker/breaker.h"
 #include "driver/pipeline/control.h"
-#include "nlohmann/json.hpp" 
+#include "nlohmann/json.hpp"
 #include "driver/task/task.h"
 #include "driver/config/config.h"
 #include "driver/ni/error.h"
 #include <condition_variable>
 #include "ts_queue.h"
+#include "driver/ni/scale.h"
```

Comment:
These file names are confusing. This file seems like it mixes types across digital readers, writers, and analog inputs. I think we could extract these into independent headers that are easier to follow and read